ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -22,6 +22,15 @@ @implementation RNSScreenStackHeaderSubview
 
 #pragma mark - Common
 
+// We're forcing the navigation controller's view to re-layout
+// see: https://github.com/software-mansion/react-native-screens/pull/2385
+- (void)ensureNavigationLayout
+{
+  UIViewController *vc = self.reactSuperview.reactSuperview.reactViewController;
+  UINavigationController *navctr = vc.navigationController;
+  [navctr.view layoutIfNeeded];
```

Comment:
Do we need to layout whole view of the navigation controller? Won't it be enough to call `navctr.navigationBar layoutIfNeeded]`?

Reply:
`navctr.navigationBar` seems enough, thanks! https://github.com/software-mansion/react-native-screens/pull/2385/commits/9179b6b6aa8afe9626c453d4d99d27b84bbd3595